### ISSUE TITLE
Reddit can not be verified

### ISSUE THREAD

**Issue by sdimkov** [keybase/keybase-issues#3311]
I'm unable to verify my Reddit account. I tried multiple times. Keybase fails at the last step (to verify my proof).

[My Keybase profile](https://keybase.io/sdimkov)
[My Reddit profile](https://www.reddit.com/user/sdimkov)
[My proof post](https://www.reddit.com/r/KeybaseProofs/comments/a55lc2/my_keybase_proof_redditsdimkov_keybasesdimkov/)

**Comment by mlsteele** [comment]
Your post on reddit has no body, only a title. Here's [an example](https://www.reddit.com/r/KeybaseProofs/comments/a55n4y/my_keybase_proof_redditruairinewman_keybaseruairi/) of a working proof. What os, browser, and version of keybase did you use to make the post? I've sometimes [seen that form drop the body](https://www.reddit.com/r/redesign/comments/8evfap/bug_post_contents_gets_dropped_when_using/) but thought we were using old.reddit. Will check it out. As a work around you could try clicking the body of the post before posting it.

**Comment by mlsteele** [comment]
I just tried making a reddit proof and the link was to old reddit (https://old.reddit.com). Even on an account opted in to the redesign. Let me know if you're ending up on a form that's not on old reddit.

**Comment by sdimkov** [comment]
Yep you're right. The problem is most likely in the redesign. I didn't realize an actual body is needed, because when I followed the provided links, the pre-filled form had no body.

Now I opted out of the redesign and retried. It worked fine.

**Comment by mlsteele** [comment]
The url Keybase shows you should be old.reddit.com which should punch through redesign preferences. So that's weird. But glad that worked for you.

**Comment by bgbrandongomez** [comment]
The link says www.reddit.com when I hovered over the submit prefilled button. When I can post again in 7 minutes, I am going to try reposting using old.reddit.com and see if that works.

**Comment by bgbrandongomez** [comment]
Using old.reddit.com works fine.

**Comment by MasterGroosha** [comment]
Why is this issue closed?
I think there should be either a note on Keybase to use old.reddit.com or form template should be changed to support both designs.

**Comment by mlsteele** [comment]
@SenorContento what platform are you using @SenorContento that you saw www.reddit.com on hover? And what were you hovering over, was it in the Keybase app, website?

**Comment by mlsteele** [comment]
I should've read the timestamps on those messages... @MasterGroosha are you seeing a link to non-old reddit?

**Comment by MasterGroosha** [comment]
@mlsteele This link leads to `https://www.reddit.com/r/KeybaseProofs/submit...`
![image](https://user-images.githubusercontent.com/2780044/55915308-125feb00-5bf2-11e9-9b6c-bec3dfeeda9b.png)
And when I follow this link, "text" field is empty, even though URL has all the necessary text in it
![image](https://user-images.githubusercontent.com/2780044/55915376-46d3a700-5bf2-11e9-9a56-8286adeba656.png)

**Comment by SenorContento** [comment]
@mlsteele I was using Chrome on Mac OSX.

**Comment by racuna** [comment]
Same problem, tried twice. And then tried again on the phone. No luck either.

**Comment by mlsteele** [comment]
@racuna using the website not the app?

@racuna, @SenorContento what does "Reddit form prefilled" link to? Does any redirect happen after you click it?

**Comment by racuna** [comment]
mine: https://www.reddit.com/r/KeybaseProofs/comments/bemkg1/my_keybase_proof_redditracunix_keybaseracuna/

Thenk I switched to old reddit and worked.

**Comment by user00265** [comment]
Well, I recently saw that my Reddit proof was no longer valid (post was still there), so I tried re-doing it... I have tried several times now and even though it looks like the others... it still doesn't work.

Keybase username: eresto
Reddit username: user00265
